src/Aspire.Dashboard/Extensions/FluentUIExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Aspire.Dashboard.Extensions;
 
-internal static class FluentUIExtensions
+internal static class FluentUiExtensions
```

Comment:
In .NET, two-letter acronyms are capitalized. So this change makes the name non-idiomatic.

Was there a reason for this change? If so, can we come up with a different name? If you really want to keep this name then the file name should be changed to match it.

Reply:
I was not aware of the convention. Reverted.